service/src/models/archaeologist.ts

```diff
@@ -148,17 +148,23 @@ export class Archaeologist {
                 maxRewrapInterval,
                 diggingFee,
                 timestamp,
-              }: SarcophagusParameters = JSON.parse(new TextDecoder().decode(data));
+              }: SarcophagusNegotiationParams = JSON.parse(new TextDecoder().decode(data));
+
+              const maximumRewrapIntervalBN = BigNumber.from(maxRewrapInterval);
+
               let error: number | null = null;
-              if (maxRewrapInterval > inMemoryStore.profile!.maximumRewrapInterval.toNumber()) {
+              if (maximumRewrapIntervalBN.gt(inMemoryStore.profile!.maximumRewrapInterval)) {
                 error = SarcophagusValidationError.MAX_REWRAP_INTERVAL_TOO_LARGE;
               }
-              if (BigNumber.from(diggingFee).lt(inMemoryStore.profile!.minimumDiggingFee)) {
+
+              if (ethers.utils.parseEther(diggingFee).lt(inMemoryStore.profile!.minimumDiggingFee)) {
```

Comment:
Is diggingFee being sent in as an ether value and not a wei value from the webapplication? We might consider making it the standard that we always deal in a BigNumber of wei since the profile on the archaeologist memory store is using wei